.circleci/config.yml

```diff
@@ -3,25 +3,25 @@ version: 2.1
 executors:
   node-browsers-small:
     docker:
-      - image: cimg/node:20.17-browsers
+      - image: cimg/node:20.12.0-browsers
     resource_class: small
     environment:
       NODE_OPTIONS: --max_old_space_size=2048
   node-browsers-medium:
     docker:
-      - image: cimg/node:20.17-browsers
```

Comment:
20.17 doesn't match `ubuntu-2404:2024.05.1` (aka `ubuntu-2404:current`), so i've updated to so the versions match.